inscriptions/ISic001137.xml

```diff
@@ -190,29 +195,19 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140622">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#MH">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Βαλ
-                    <supplied reason="lost">ερ</supplied>
-                    <supplied reason="omitted">ί</supplied>
-                    <expan>
-                        <abbr>α</abbr>
-                        <ex>?</ex>
-                    </expan>
-                    <lb n="2"/>ἡ τὸν βίο<supplied reason="lost">ν εὐ</supplied>-<lb n="3"/>πρεπ<supplied reason="omitted">ῶ</supplied>ς β<supplied reason="lost">ιώσα</supplied>-<lb n="4"/>σα <unclear>
-                    ἔ</unclear>ζη
-                    <supplied reason="lost">σεν ἔτη —ʹ</supplied>
-                    <lb n="5"/>
+                    <lb n="1"/><name>Βαλ<supplied reason="lost">ερ</supplied><choice><corr>ί</corr><sic>k</sic></choice>α</name> <gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
very much the right idea with the corr/sic, but, annoyingly, not in this case. This is one of the awkward cases where the antiquarian tradition does its best reading something difficult, but then Kaibel has in turn done his best to make sense of it. What is best is to present Kaibel's edited text, and then present any alternatives for individual letters simply as notes in the apparatus, because there is no reason actually to think that e.g. 'K' was what wsa originally (mistakenly?) cut on the stone, only that this was the 17th century reader's best effort to interpret the remaining traces.
and so on for the remaining lines.